reference/5.1/Microsoft.PowerShell.Utility/Compare-Object.md

```diff
@@ -316,17 +389,25 @@ You can send an object down the pipeline to the **DifferenceObject** parameter.
 
 ### None
 
-If the **reference** object and the **difference** object are the same, there's no output.
+If the **reference** object and the **difference** object are the same, there's no output, unless
+you use the **IncludeEqual** parameter.
 
 ### System.Management.Automation.PSCustomObject
 
 If the objects are different, `Compare-Object` wraps the differing objects in a `PSCustomObject`
-wrapper with a **SideIndicator** property to reference the differences. When you use the
-**PassThru** parameter, `Compare-Object` omits the `PSCustomObject` wrapper around the compared
-objects and returns the differing objects, unchanged.
+wrapper with a **SideIndicator** property to reference the differences.
+
+When you use the **PassThru** parameter **Type** of the object is not changed but the instance of
+the object returned has an added **NoteProperty** named **SideIndicator**. **SideIndicator** shows
+which input object the output belongs to.
 
 ## NOTES
 
+When using the **PassThru** parameter, the output displayed in the console may not include the
+**SideIndicator** property. The default format view of the for the object type output by
+`Compare-Object` does not include the **SideIndicator** property. For more information see Example 5
```

Comment:
The see Example 5 should be a link to example 5.

Reply:
OPS does not allow bookmark links to H3 so I added an anchor tag.

Reply:
I bookmark to H3s all the time... I don't think that's an accurate statement.